_attributes/common-attributes.adoc

```diff
@@ -49,7 +49,7 @@ endif::openshift-origin[]
 :rh-storage: OpenShift Data Foundation
 :rh-rhacm-first: Red Hat Advanced Cluster Management (RHACM)
 :rh-rhacm: RHACM
-:rh-rhacm-version: 2.5
+:rh-rhacm-version: 2.8
```

Comment:
@rohennes is there a reason why this PR only targets 4.13 for this change (instead of targeting main, and then hitting both 4.13 and 4.14)? 

Doesn't seem like we'd want to up the version in 4.13, but leave the older version in 4.14+